src/som/interpreter/objectstorage/ObjectTransitionSafepoint.java

```diff
@@ -10,6 +10,7 @@
 import com.oracle.truffle.api.nodes.InvalidAssumptionException;
 
 import som.compiler.MixinDefinition.SlotDefinition;
+import som.vm.CustomPhaser;
```

Comment:
CustomPhaser, not a great name.
How about: `SafepointPhaser`?

Reply:
good idea